ZombieSharp/Statistic.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Dapper;
+using Microsoft.Extensions.Logging;
+using MySql.Data.MySqlClient;
+using Newtonsoft.Json;
+
+namespace ZombieSharp
+{
+    public partial class ZombieSharp
+    {
+        private StatsDatabase statsConfig;
+        MySqlConnection connection = null;
+
+        private void StatsOnLoad()
+        {
+            var statsConfigPath = Path.Combine(ModuleDirectory, $"../../configs/zombiesharp/database.json");
+            statsConfig = JsonConvert.DeserializeObject<StatsDatabase>(File.ReadAllText(statsConfigPath));
+            connection = StatsConnect();
+        }
+
+        private MySqlConnection StatsConnect()
+        {
+            var query = $"Server={statsConfig.hostname}; database={statsConfig.database}; UID={statsConfig.username}; password={statsConfig.password}";
```

Comment:
Is it possible for now to get connection from DI and not manually resolve it in each service? It's better to re-use connections for better performance instead of creating new for each service.
https://mysqlconnector.net/tutorials/dapper/#4-create-your-connection